### PR TITLE
Check if Jenkins is quieting down.

### PR DESCRIPTION
**Pull request by christ66**
@reviewbybees

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
🐝

**Comment by aheritier** [comment]
🐝

File: src/main/java/com/cloudbees/jenkins/support/impl/BuildQueue.java [hunk 1]
```diff
@@ -101,6 +101,7 @@ public void writeTo(OutputStream os) throws IOException {
               out.println("----");
               out.println();
             }
+            out.println("Is quieting down: " + Helper.getActiveInstance().isQuietingDown());
```
Comment (carlosrodlop) [inline src/main/java/com/cloudbees/jenkins/support/impl/BuildQueue.java]: 🐝 
Suggestion:
out.println("Is quieting down: " + Helper.getActiveInstance().isQuietingDown() + "new builds are not being executed");

**Comment by jglick** [comment]
BTW you could take the opportunity to correct the missing information noted in [JENKINS-38514](https://issues.jenkins-ci.org/browse/JENKINS-38514).

**Pull request merged**